pkg/cloud-provider/cloudapi/aws/aws_cloudinterface_impl.go

```diff
@@ -105,14 +105,14 @@ func (c *awsCloud) GetAccountStatus(accNamespacedName *types.NamespacedName) (*c
 	return c.cloudCommon.GetStatus(accNamespacedName)
 }
 
-// AddInventoryPoller adds poller for polling cloud inventory.
-func (c *awsCloud) AddInventoryPoller(accountNamespacedName *types.NamespacedName) error {
-	return c.cloudCommon.AddInventoryPoller(accountNamespacedName)
+// DoInventoryPoll calls cloud API to get vm and vpc resources.
```

Comment:
just cloud resources is fine.. later we might poll interfaces or sg.. so we dont want to keep updating the comment